composer.json

```diff
@@ -4,6 +4,16 @@
     "homepage": "http://vfs.bovigo.org/",
     "description": "Virtual file system to mock the real file system in unit tests.",
     "license": "BSD-3-Clause",
+    "authors": [
+        {
+          "name": "Frank Kline",
+          "homepage": "http://frankkleine.de/"
+        }
+    ],
+    "support": {
+        "issues": "https://github.com/sndsgd/sndsgd-util/issues",
+        "source": "https://github.com/sndsgd/sndsgd-util/tree/master"
```

Comment:
Maybe both should point to the vfsStream repository - or would you like to receive all vfsStream issues in your repository? ;-)